apps/web/src/lang/fr.json

```diff
@@ -6590,6 +6542,54 @@
     "defaultMessage": "Créer une nouvelle équipe à partir de zéro",
     "description": "Descriptive text for the create team page in the admin portal."
   },
+  "05m1mY": {
+    "defaultMessage": "Modifier les informations de l’équipe",
+    "description": "Title for the team edit page"
+  },
+  "9i2N/g": {
+    "defaultMessage": "Modifier le compte utilisateur",
+    "description": "Title for the user edit page"
+  },
+  "Cjp2F6": {
+    "defaultMessage": "Informations sur le bassin de candidats",
+    "description": "Title for the pool info page"
+  },
+  "Rl+0Er": {
+    "defaultMessage": "Afficher les candidats",
+    "description": "Title for the edit pool page"
+  },
+  "SFk84j": {
+    "defaultMessage": "Informations sur l’utilisateur",
+    "description": "Link text for the user information page link"
+  },
+  "SLedtO": {
+    "defaultMessage": "Profil de l’utilisateur",
+    "description": "Title for the user profile page"
+  },
+  "UHH1Oh": {
+    "defaultMessage": "Informations sur l’équipe",
```

Comment:
As I understand it, `informations` is the generic term, while `renseignements` is situationally specific to information that you are seeking out to learn. Because this is referring to the information that exists on the page, rather than "I am going to look at team information", I think `informations` is correct.